docs/modules/demos/pages/jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data.adoc

```diff
@@ -141,78 +124,15 @@ image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_run_noteb
-This is described below.
-
-=== Install the libraries into a product image
-
-Libraries can be added to a custom *product* image launched by the notebook. Suppose a Spark job is prepared like this:
-
-// TODO (@NickLarsenNZ): Use stackable0.0.0-dev so that the demo is reproducable for the release
-// and it will be automatically replaced for the release branch.
-// Also update the reference in notebook.ipynb.
-
-[source,python]
-----
-spark = (SparkSession
-            .builder
-            .master(f'k8s://https://{os.environ["KUBERNETES_SERVICE_HOST"]}:{os.environ["KUBERNETES_SERVICE_PORT"]}')
-            .config("spark.kubernetes.container.image", "oci.stackable.tech/stackable/spark-k8s-with-scikit-learn:3.5.0-stackable24.3.0")
-            .config("spark.driver.port", "2222")
-            .config("spark.driver.blockManager.port", "7777")
-            .config("spark.driver.host", "driver-service.default.svc.cluster.local")
-            .config("spark.driver.bindAddress", "0.0.0.0")
-            .config("spark.kubernetes.authenticate.driver.serviceAccountName", "spark")
-            .config("spark.kubernetes.authenticate.serviceAccountName", "spark")
-            .config("spark.executor.instances", "4")
-            .config("spark.kubernetes.container.image.pullPolicy", "IfNotPresent")
-            .appName("taxi-data-anomaly-detection")
-            .getOrCreate()
-        )
-----
-
-It requires a specific Spark image:
-
-// TODO (@NickLarsenNZ): Use stackable0.0.0-dev so that the demo is reproducable for the release
-// and it will be automatically replaced for the release branch.
-// Also update the reference in notebook.ipynb.
-
-[source,python]
-----
-.config("spark.kubernetes.container.image",
-  "oci.stackable.tech/stackable/spark-k8s-with-scikit-learn:3.5.0-stackable24.3.0"),
-...
-----
-
-This is created by taking a Spark image, in this case `oci.stackable.tech/sdp/spark-k8s:3.5.0-stackable24.3.0`, installing specific python libraries into it
-, and re-tagging the image:
-
-[source,console]
-----
-FROM oci.stackable.tech/sdp/spark-k8s:3.5.0-stackable24.3.0
-
-COPY demos/jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/requirements.txt .
-
-RUN pip install --no-cache-dir --upgrade pip && \
-    pip install --no-cache-dir -r ./requirements.txt
-----
-
-Where `requirements.txt` contains:
-
-[source,console]
-----
-scikit-learn==1.3.1
-pandas==2.0.3
-----
-
-NOTE: Using a custom image requires access to a repository where the image can be made available.
+The Python notebook uses libraries such as `pandas` and `scikit-learn` to analyze the data.
+In addition, since the model training is delegated to a Spark Connect server, some of these dependencies, most notably `scikit-learn`, must also be made available on the Spark Connect pods.
+For convenience, a custom image is used in this demo that bundles all the required libraries for both the notebook and the Spark Connect server.
```

Comment:
optional: We could link to the Dockerfile (so that others can take the next steps for their use case).